### ISSUE TITLE
No context menu border on Firefox

### ISSUE THREAD

**Issue by Tromzy** [vinceliuice/Qogir-theme#83]
Hi,

When using Qogir application theme, on Firefox my right click menu gets no border nor shadow, see here : 

https://i.imgur.com/o5dmCMA.png

The right click menu is circled in red, you can see no border / no shadow.

**Comment by vinceliuice** [comment]
Yes, because it'll have double border on gnome desktop so I removed menu border of firefox, then on other desktop it'll have no border
